compound.py

```diff
@@ -197,7 +200,9 @@ def update_bot_order_volumes(
         )
     db.commit()
 
-    new_bo = round(new_base_order_volume, rounddigits)
+    #### Instead of rounding, truncate the last values from the digits, so we don't accidentally round up
+    factor = 10.0 ** rounddigits
+    new_bo = math.trunc(new_base_order_volume * factor) / factor
```

Comment:
Deze opmerking gaat niet over dit stukje. Zie mijn opmerking op Discord; de BO naar beneden wijzigen gaat niet bij negatieve profit door de if op regel 210. Zouden we van die if een 'if new_bo == base_order_volume' kunnen maken? Immers, als door de truncate hetzelfde getal naar 3C gaat hebben we te weinig profit (zowel positief als negatief) om de bot te updaten en willen we dat opslaan (ook zowel positief als negatief).

In alle andere gevallen, de else case, is de truncated value anders dan wat de bot in 3C heeft en kan de update dus plaatsvinden.

Reply:
Volgens mij fixed. Hij lijkt ook naar beneden te compounden nu